bokehjs/test/integration/glyphs/rect.ts

```diff
@@ -27,4 +28,17 @@ describe("Rect glyph", () => {
 
     await display(row([make_plot("canvas"), make_plot("svg"), make_plot("webgl")]))
   })
+
+  it("should support adding new glyph", async() => {
```

Comment:
Looking at the code, it's about changing data not glyphs? Suppose it should be adding a new data point to an existing glyph.

Reply:
Yes, it is about changing the count of a single glyph. I'll change the `it` description.